src/backend/db/categories.js

```diff
@@ -8,50 +8,50 @@ import { v4 as uuid } from "uuid";
 export const categories = [
   {
     _id: uuid(),
-    categoryName: "CRICKET BATS",
+    categoryName: "BAT",
     img: "https://res.cloudinary.com/dgqwptcvp/image/upload/v1647921594/CricBazzar%20Ecommerce/Cricket-bat_lwscgk.jpg",
-    description: "",
+    description: "BATS",
   },
   {
     _id: uuid(),
-    categoryName: "CRICKET BALLS",
+    categoryName: "BALL",
     img: "https://res.cloudinary.com/dgqwptcvp/image/upload/v1647921638/CricBazzar%20Ecommerce/cricket-ball_lco1zt.jpg",
-    description: "",
+    description: "BALLS",
   },
   {
     _id: uuid(),
-    categoryName: "CRICKET GLOVES",
+    categoryName: "PROTECTION",
     img: "https://res.cloudinary.com/dgqwptcvp/image/upload/v1647921655/CricBazzar%20Ecommerce/cricket-gloves_jswvvp.jpg",
-    description: "",
+    description: "GLOVES",
   },
   {
     _id: uuid(),
-    categoryName: "CRICKET PADS",
+    categoryName: "PROTECTION",
     img: "https://res.cloudinary.com/dgqwptcvp/image/upload/v1647921662/CricBazzar%20Ecommerce/cricket-pads_qjpl23.jpg",
-    description: "",
+    description: "PADS",
   },
   {
     _id: uuid(),
-    categoryName: "CRICKET CLOTHINGS",
+    categoryName: "CLOTHINGS",
     img: "https://res.cloudinary.com/dgqwptcvp/image/upload/v1647921646/CricBazzar%20Ecommerce/cricket-clothing_mh8hsx.jpg",
-    description: "",
+    description: "CLOTHINGS",
   },
   {
     _id: uuid(),
-    categoryName: "CRICKET SHOES",
+    categoryName: "SHOES",
     img: "https://res.cloudinary.com/dgqwptcvp/image/upload/v1647921629/CricBazzar%20Ecommerce/cricket_shoe_v3mnpp.jpg",
-    description: "",
+    description: "SHOES",
   },
   {
     _id: uuid(),
-    categoryName: "PROTECTION GEAR",
+    categoryName: "PROTECTION",
     img: "https://res.cloudinary.com/dgqwptcvp/image/upload/v1647921671/CricBazzar%20Ecommerce/cricket-protection-kit_o6obky.png",
-    description: "",
+    description: "PROTECTION GEAR",
   },
   {
     _id: uuid(),
     categoryName: "ACCESSORIES",
     img: "https://res.cloudinary.com/dgqwptcvp/image/upload/v1647921622/CricBazzar%20Ecommerce/cricket_bags_r7olq7.jpg",
-    description: "",
+    description: "ACCESSORIES",
```

Comment:
Its just a suggestion : you can provide better name instead of description here. as it is one word. like you can give img_tag, tagline...

Reply:
Yeah will do that